### ISSUE TITLE
Checkbox component is not focusable and therefore not accessible without a mouse

### ISSUE THREAD

**Issue by snapsnapturtle** [freenowtech/wave#23]
The checkbox item can't be focused and therefore cannot be used without the need of a mouse.
